packages/device_info_plus/device_info_plus_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.1+0
+
+- breaking change: toMap renamed as toJson for dart:convert compatibility
```

Comment:
message outdated, the toMap should not be renamed to toJson

Reply:
I will update it in a couple of days, sorry about that (I'm a bit stuffed)

Reply:
fixed in [a880118](https://github.com/fluttercommunity/plus_plugins/pull/950/commits/a8801184ce028c19f23c8e54443391a7723b0c07)